src/daemon.py

```diff
@@ -2,10 +2,11 @@
 """
 Contains code for the message bus daemon.
 """
-from mycroft.api import DeviceApi, check_remote_pairing
-from mycroft.util.signal import create_signal
-from mycroft_bus_client import Message
-from mycroft_bus_client import MessageBusClient as OriginalMessageBusClient
+from ovos_backend_client.api import DeviceApi
+from ovos_backend_client import check_remote_pairing
+from ovos_utils.signal import create_signal
```

Comment:
note: signal utils are in the process of being deprecated in ovos, nothing is really consuming these anymore except for maybe the mk1 integration (which is broken and under rewrite anyway)

the call create_signal can be completely removed